### DISCUSSION TITLE
He3 剪切板功能讨论

### DISCUSSION THREAD

**Discussion by lyzhang1999** [he3-app/he3-feedback#128] [General]
计划功能：
- 文件预览
- 剪切板内容识别和工具推荐
- 云上存储
- 剪切板历史
- 图片预览

如果对剪切板功能有更多诉求，欢迎提出你的建议。

**Comment by lyzhang1999** [comment]
@liuxzzz

**Comment by rockpanda** [comment]
时间戳筛选、多终端同步、快捷回复

**Reply by lyzhang1999** [reply]
感谢建议~

**Reply by liuxzzz** [reply]
> 时间戳筛选、多终端同步、快捷回复

你好，请问所说的快捷回复指的是什么呢？
